runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -328,6 +328,12 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
          */
         boolean cacheWrites = true;
 
+        /**
+         * Used only by LR.
+         * On LogData.getPayload(), after deserializing the data part, the data part should not be set to null.
```

Comment:
Add a comment about OpaqueStream relying on the serialized form and failing for this case.